cli/README.md

```diff
@@ -1,6 +1,6 @@
-# redoc-cli
+# Redoc-cli
 
-**[ReDoc](https://github.com/Redocly/redoc)'s Command Line Interface**
+Redoc CLI is an open source tool that provides linting against a customizable ruleset, as well as bundling of the OpenAPI files into a single file. You can also preview the output of your docs with the Redocly API References through this tool.
```

Comment:
This sounds a bit confused with https://github.com/Redocly/openapi-cli — the redoc-cli is just the interface of the main Redoc, bundling/serving/previewing the output, so I'd rather not describe any "business" features here and keep the link to the repo root itself instead for more explanation (where such maintenance is provided, w/ up to date information).

Reply:
Well,  `redoc-cli `https://github.com/Redocly/redoc/tree/master/cli and `openapi-cli`   https://github.com/Redocly/openapi-cl are different. The readme for the `redoc-cli` was sparse so I added more information in it.

Reply:
IMO the ideal way would be to describe the difference that it makes from core redoc js, which is to "bundle your docs into zero-dependency HTML file", i. e. for a self-contained output.

Reply:
… and, yes, it's a live (`--watch`) preview for any spec changes in real time, and also the way of server-side React pre-rendering, so it's kinda hard to describe just on its own, I agree;)

Maybe that's why the current docs are so sparse, as it serves several use cases for development, production and distribution that are inferred from other parts of the docs (or GH issues for that matter).

Reply:
Thanks for this

Reply:
I will try to draw a definition from the main documentation, thanks alot for your comments